extra/1-factorial.js

```diff
@@ -10,6 +10,16 @@
 
```

Comment:
Looks like there's a lot going on here 😄 
For problems like this, I would suggest we take a step back from the code - and think about some of the Problem Solving techniques we covered in the workshop in class. Maybe start by thinking about how you would normally do something like this (without a computer) and try drawing it out on a whiteboard or paper. This may give you some ideas about how to do this in code.

Reply:
Well done for attempting this though! It's a tough one!

Reply:
I wrote pseudo code and watched your videos on function and looping and managed to do this on my own in less 30 minutes. Thank you, Barath!